src/hair/hair.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./hairStyle.css">
+  <link rel="stylesheet" href="reset.css">
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
+  <title>Snazzy Hair Salon</title>
+</head>
+
+<body>
+  <aside>
+    <div class="logoimg">
+      <!-- with logo img as background in css -->
+    </div>
+    <!-- blank row -->
+    <nav id="navabout">
+      <h3>ABOUT</h3>
+    </nav>
+    <nav id="navmeet">
+      <h3>MEET</h3>
+    </nav>
+    <nav id="navservices">
+      <h3>SERVICES</h3>
+    </nav>
+    <nav id="navbootcamp">
+      <h3>BOOTCAMP</h3>
+    </nav>
+    <nav id="navproducts">
+      <h3>PRODUCTS</h3>
+    </nav>
+    <nav id="navlatest">
+      <h3>LATEST</h3>
+    </nav>
+    <nav id="navgiftcards">
+      <h3>GIFT CARDS</h3>
+    </nav>
+    <nav id="navcontact">
+      <h3>CONTACT</h3>
+    </nav>
```

Comment:
add some padding to these nav elements. text touching the edge of the container (and in this case, the screen) is a no-no. 20px is a good number to start with. `padding-left: 20px;`